### ISSUE TITLE
Solid edge but no study ??

### ISSUE THREAD

**Issue by jar398** [OpenTreeOfLife/feedback#287]
The edge to Ophioglossidae is solid, i.e. supported by a study, but when selected, no supporting study is listed.
# 

| Metadata | Do not edit below this line |
| :-- | :-- |
| Author | [Jonathan A Rees](https://github.com/jar398) |
| Upvotes | 0 |
| URL | [tree.opentreeoflife.org/opentree/argus/opentree5.0@mrcaott3223ott93156/Equisetidae--Ophioglossidae](https://tree.opentreeoflife.org/opentree/argus/opentree5.0@mrcaott3223ott93156/Equisetidae--Ophioglossidae) |
| Target node label | [Equisetidae + Ophioglossidae] |
| Synthetic tree id | opentree5.0 |
| Synthetic tree node id | mrcaott3223ott93156 |
| Source tree id(s) |  |
| Open Tree Taxonomy id |  |
| Supporting reference | None |

**Comment by mtholder** [comment]
I'm seeing a link to https://tree.opentreeoflife.org/curator/study/view/pg_2820?tab=trees&tree=tree6566 in the properties panel, when I click on the "i" for that group.

**Comment by jar398** [comment]
I don't know what you're doing, but I've been able to reproduce this in
three different web browsers. Follow the URL given in this feedback issue,
then click on the blue 'i' for the Ophioglossidae node or edge. No study is
shown in the properties panel.

**Comment by mtholder** [comment]
Yeah. I'm not seeing it now either. I think that I was seeing properties for the parent node, and thought that I had clicked on Ophioglossidae, but had actually been a little off.  On Chrome on Ubuntu, I see a phenomenon in which the blue 'i' appears when I mouse over a branch, and it disappears when my cursor is far enough away from the branch. But there is an unhappy intermediate state: I have moused over to make the 'i' visible, but I am currently pointing just a little off of the 'i' button. In that state clicks have no effect (so if you had the properties panel up with a previous node's properties it is easy to mistake the properties panel as being relevant to the node that you just clicked near).

tl;dr version: I think @jar398  is correct and it was user error on my part, and we may want to revisit the interaction of click-target behavior and appearance/disappearance of the blue 'i'

**Comment by mtholder** [comment]
closing. I think the solid lines needing support has been fixed (regardless, the node indicated in this issue is not an example of this in the more recent trees, so the issue is less helpful).
